### PR TITLE
configure eslint and husky

### PR DESCRIPTION
**Pull request by rana123bilal**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "settings": {
+    "react": {
+      "pragma": "React",
+      "version": "16.13.1"
```
Comment (Vladlan) [inline .eslintrc.json:18]: which version of React are you using?

**Review by Vladlan — APPROVED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (rana123bilal) [reply .eslintrc.json:18]: I am using react 18.2.0.

Comment (Vladlan) [reply .eslintrc.json:18]: @rana123bilal than why it is 16.13.1

Comment (rana123bilal) [reply .eslintrc.json:18]: I already updated it and merged this PR.

**Pull request merged**